spec/quiet_quality/version_control_systems/git_spec.rb

```diff
@@ -10,39 +10,121 @@
   end
 
   context "instance methods" do
-    let(:path) { "/path/to/git/repo" }
-    let(:instance) { described_class.new(path) }
+    let(:git_repo_path) { "tmp/repo" }
+    let(:base_sha) { "main" }
 
-    let(:remote_double) { instance_double(Git::Remote, url: "remote_url") }
-    let(:git_double) { instance_double(Git::Base, remote: remote_double, merge_base: [expected_base]) }
-    let(:expected_base) { instance_double(Git::Object::Commit, sha: "abcd1234") }
+    let(:instance) { described_class.new(git_repo_path) }
```

Comment:
You've made the base context here use the fixture; would you mind deleting my context at the bottom? I initially added an extra context for the fixture-backed testing so I could leave your existing tests alone :-)